src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,24 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int START_YEAR = 0;
```

Comment:
```suggestion
    private static final int START_YEAR_INDEX = 0;
```
The one who doesn't know the task requirement would think that start year is 0, like 2024 years ago